ARrehab/ARrehab/Tile.swift

```diff
@@ -13,11 +13,13 @@ import CoreGraphics
 class Tile : Entity, HasModel, HasCollision {
     
     //Class attributes
-    static let TILE_SIZE = SIMD3<Float>(0.5, 0.01, 0.5)
+    static let SCALE: Float = 1
+    static let TILE_SIZE = SIMD3<Float>(0.5 / SCALE, 0.01, 0.5 / SCALE)
     static let TILE_COLLISION_GROUP = CollisionGroup(rawValue: 1) //Totally arbitrary number
    
     static let defaultTileModel = ModelComponent(mesh: MeshResource.generateBox(size: Tile.TILE_SIZE, cornerRadius: 0.2), materials: [SimpleMaterial()])
```

Comment:
I suggest instead of scaling each individual tile like so, we should scale the board as a whole post generation.
What I mean by this is generate the entire board as normal, then scale the parentEntity of all these tiles. This can be done in TileGrid.swift

Reply:
#22